src/test/resources/testConfig.conf

```diff
@@ -0,0 +1,5 @@
+input.osm.pbf.file = "input_file_path"
+input.asset.file.directory = "asset_directory"
+input.asset.file.hierarchic = false
+output.file.directory = "output_file_path"
+generation.lv.distinctHouseConnections = true
```

Comment:
Differs from default? By purpose?

Reply:
Yes to check, that not the default value is used and the config is parsed correctly.